vulnerable_people_form/form_pages/shared/location_tier.py

```diff
@@ -22,8 +22,12 @@ def update_location_tier_by_uprn(uprn, app):
     if app.is_tiering_logic_enabled:
         location_tier = get_uprn_tier(uprn)
         set_location_tier(location_tier)
+
         lad_code = get_ladcode_from_uprn(uprn)
```

Comment:
does it make sense to move this into a method as its the same logic in 2 places?

Reply:
Maybe, but I'm not that keen to refactor unless it starts getting used more widely.